README.md

```diff
@@ -249,6 +249,7 @@ The following table lists the parameters for the `deployments` component and the
 | `deployments.probesOverrides.failureThreshold` | Override the `failureThreshold` for every Readiness and Liveness probes. | `nil` |
 | `deployments.mongodbExistingSecret` | Use a different MongoDB secret for this service | `nil` |
 | `deployments.minReadySeconds` | Wait before considering the new pod available | `nil` |
+| `deployments.storageMaxImageSize` | Maximum Artifacts image size in bytes. | `10737418240` |
```

Comment:
Should we also add `deployments.storageMaxGenerateDataSize` (`DEPLOYMENTS_STORAGE_MAX_GENERATE_DATA_SIZE`) for generating single file artifacts (when you upload a file in the UI that is not an artifact).

Reply:
There you go: https://github.com/mendersoftware/mender-helm/pull/484/commits/da4bc9a96762abe98ddf481e0d9d455bfaa16df4